### PR TITLE
Update doc routes, center all the docs examples

### PR DESCRIPTION
**Pull request by mcnuttandrew**

### CODE REVIEW

File: docs/hint.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ## Hint
```
Comment (mcnuttandrew) [inline docs/hint.md]: Bizzarely there's never been any of our many hint example on this page!

File: website/src/mdRoutes.js [hunk 2]
```diff
@@ -1,65 +1,63 @@
-import presentation from '../../docs/presentation.md';
```
Comment (mcnuttandrew) [inline website/src/mdRoutes.js]: i was trying to figure out where to put highlight and hexbin and I rage quit trying to mentally diff the whole thing

File: docs/hint.md (hunk 1 above)
Comment (benshope) [reply docs/hint.md]: Seems good to add one

File: website/package.json [hunk 3]
```diff
@@ -24,6 +24,6 @@
     "d3-force": "^1.0.6",
     "d3-random": "^1.1.0",
     "react-virtualized": "^9.18.5",
-    "react-vis": "^1.9.2"
+    "react-vis": "^1.11.2"
```
Comment (benshope) [inline website/package.json]: nice!  I wonder how this got out of date - perhaps bumping this should also be part of a script

**Pull request merged**